MAUI/Cartesian-Charts/Errorbar-Chart.md

```diff
@@ -0,0 +1,762 @@
+                          HorizontalErrorValue="0.5"
+                          VerticalDirection="Both">  
+    </chart:ErrorBarSeries>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+    ErrorBarSeries errorBar = new ErrorBarSeries()
+    {
+        ItemsSource = new ViewModel().EnergyProductions,
+        XBindingPath = "ID",
+        YBindingPath = "Coal",
+        HorizontalErrorValue = 0.5,
+        VerticalErrorValue = 50,
+        VerticalDirection="Both"
+    };
+    chart.Series.Add(series);
+    chart.Series.Add(errorBar);
+
+
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Vertical Direction as Both in Error Bar Series](Chart-types_images/BasicRenderingErrorBar.png)
+
+#### Plus
+
+The following code illustrates how to set the [VerticalDirection]() value as Plus.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+    <chart:ErrorBarSeries ItemsSource="{Binding EnergyProductions}"
+                          XBindingPath="ID"
+                          YBindingPath="Coal"
+                          VerticalErrorValue="50"
+                          HorizontalErrorValue="0.5"
+                          VerticalDirection="Plus">    
+    </chart:ErrorBarSeries>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+    ErrorBarSeries errorBar = new ErrorBarSeries()
+    {
+        ItemsSource = new ViewModel().EnergyProductions,
+        XBindingPath = "ID",
+        YBindingPath = "Coal",
+        HorizontalErrorValue = 0.5,
+        VerticalErrorValue = 50,
+        VerticalDirection="Plus"
+    };
+    chart.Series.Add(series);
+    chart.Series.Add(errorBar);
+
+
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Vertical Direction as Plus in Error Bar Series](Chart-types_images/VerDirectionPlus.png)
+
+#### Minus
+
+The following code illustrates how to set the [VerticalDirection]() value as Minus.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+    <chart:ErrorBarSeries ItemsSource="{Binding EnergyProductions}"
+                          XBindingPath="ID"
+                          YBindingPath="Coal"
+                          VerticalErrorValue="50"
+                          HorizontalErrorValue="0.5"
+                          VerticalDirection="Minus">
+    </chart:ErrorBarSeries>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+    ErrorBarSeries errorBar = new ErrorBarSeries()
+    {
+        ItemsSource = new ViewModel().EnergyProductions,
+        XBindingPath = "ID",
+        YBindingPath = "Coal",
+        HorizontalErrorValue = 0.5,
+        VerticalErrorValue = 50,
+        VerticalDirection="Minus"
+    };
+    chart.Series.Add(series);
+    chart.Series.Add(errorBar);
+
+
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Vertical Direction as Minus in Error Bar Series](Chart-types_images/VerDirectionMinus.png)
+
+
+## Type
+
+SfCartesianChart supports the following type of error bar series.
+
+*`Fixed`
+
+*`Percentage`
+
+*`Standard Error`
+
+*`Standard Percentage`
+
+N> The default error bar series Type is Fixed.
```

Comment:
Mention error values should be set if the type is not custom

Reply:
I resolved this issue